salt/metalk8s/orchestrate/upgrade/init.sls

```diff
@@ -39,6 +39,15 @@ Wait for pillar refresh to complete:
 {%- set cp_nodes = salt.metalk8s.minions_by_role('master') | sort %}
 {%- set other_nodes = pillar.metalk8s.nodes.keys() | difference(cp_nodes) | sort %}
 
+Ensure all apiservers serve a certificate for 127.0.0.1:
+  salt.state:
+    - tgt: {{ cp_nodes | join(",") }}
+    - tgt_type: list
+    - sls:
+      - metalk8s.internal.upgrade.apiserver-cert-localhost
+    - saltenv: {{ saltenv }}
+    - batch: 1
```

Comment:
I think this state should require `module: Wait for pillar refresh to complete`

Reply:
I'll do it, but could you explain why?

Reply:
Because we want the pillar to be refreshed before calling this, as you use information from the pillar and as just before this highstate we just upgrade the bootstrap our pillar may-be empty